src/shogun/mathematics/linalg/ratapprox/logdet/opfunc/LogRationalApproximationCGM.h

```diff
@@ -55,14 +51,9 @@ class CLogRationalApproximationCGM : public CRationalApproximation
 	virtual ~CLogRationalApproximationCGM();
 
 	/**
-	 * method that creates a scalar job result aggregator, then creates
-	 * one job per trace sample, attaches the aggregator with them, and submits
-	 * the job to computation engine and then returns the aggregator
-	 *
-	 * @param sample the vector for which new computation jobs are to be created
-	 * @return the job result aggregator of all the jobs created
+	 * method that solves the result for a sample
 	 */
-	virtual CJobResultAggregator* submit_jobs(SGVector<float64_t> sample);
+	virtual float64_t solve(SGVector<float64_t> sample);
```

Comment:
`compute`?

Also, make sure to start the comment with a capital M. "Method to ..."